7.1 Snapshot.md

```diff
@@ -56,6 +57,36 @@ Once a message reaches the age `DELTA_PRUNE` several checks are performed, if
 + all outputs are confirmed spent, then the value message is pruned. 
 + any of the outputs are unspent, i.e. not confirmed spent, the message *shall* be stored, and kept until all outputs are confirmed spent.
 
-## 7.1.7 Post pruning
+## 7.1.3.4 Post pruning
 
 Transactions with unspent outputs at the time of pruning remain in the database. Once all their outputs are confirmed spent, i.e. the flags `confirmedSpent` are `TRUE`, the transaction is deleted.
+
+Those messages that survive the initial pruning process become part of a collection of messages called *pruning collection*. Once a transaction is confirmed spent it is removed from the pruning collection.
+
+## 7.1.4 Partial snapshots
+
+In this subsection we define partial snapshots, which create a partial view of the ledger state. This partial view can then be employed to e.g. enable syncing of a node.
+
+In this section we describe a trusted syncing mechanism.
+
+### 7.1.4.1 Epoch collections
+
+The timestamp tool *Epochs*, defined in [Section 4.2.6 - Epochs](./4.2%20Timestamps.md/#4.2.6%20Epochs), enables to group Messages objectively in groups, which we call epoch collections. 
+
+Since eventually nodes must share the same perception, nodes will reach consensus which confirmed messages are contained within a given epoch. Due to the age restriction of parents, see [4.2.3.1 Age of parents](./4.2%20Timestamps.md/#4.2.3.1%20Age%20of%20parents), no messages can be appended to an epoch once the age of the epoch is larger than `DELTA`. 
```

Comment:
Just wording suggestions
```suggestion
Since eventually nodes must share the same perception, if a given epoch is old enough, then all nodes will reach consensus about which messages are part of its epoch collection. Due to the age restriction of parents, see [4.2.3.1 Age of parents](./4.2%20Timestamps.md/#4.2.3.1%20Age%20of%20parents), no messages can be appended to an epoch once the age of the epoch is larger than `DELTA`. 
```